app/components/primer/image_crop.rb

```diff
@@ -31,6 +31,8 @@ def initialize(src:, rounded: true, **system_arguments)
       @system_arguments[:tag] = "image-crop"
       @system_arguments[:src] = src
       @system_arguments[:rounded] = true if rounded
+
+      add_js_data_attributes(@system_arguments)
```

Comment:
Not a huge fan of adding a requirement to developers that they have to remember to call this function in each behavioral component. Can we somehow do this automatically for behavioral components?

Reply:
I'm also not a huge fan, but we don't have a clear distinction of which components are behavioral, maybe we could make **all** components add the `data-view-component` attribute. We could make this change to `BaseComponent` which would reflect on all of our components.

WDYT?